lib/modules/dosomething/dosomething_reportback/dosomething_reportback.query.inc

```diff
@@ -10,10 +10,10 @@
  *
  * @param array $params
  *   An associative array of conditions to filter by. Possible keys:
- *   - nid: (string) A campaign node nid to filter by.
- *   - rbid: (string) A reportback rbid to filter by.
+ *   - nid: (string or array) A campaign node nid to filter by.
+ *   - rbid: (string or array) A reportback rbid to filter by.
  *   - random: (boolean) If set, randomly sort the results.
- *   - runs: (string) Campaign run nid(s) to filter by. 
+ *   - runs: (string or array) Campaign run nid(s) to filter by. 
```

Comment:
is this okay to write? If it's just one run_id, then it's a string. If it's multiple, it's an array.